### PR TITLE
[miio] change brightness for Philips bulbs

### PR DESCRIPTION
**Pull request by marcelrv**
add s6 vacuum

https://community.openhab.org/t/xiaomi-robot-vacuum-binding/31317/1387
https://community.openhab.org/t/xiaomi-philips-wifi-bulb-binding/36571/10

Signed-off-by: Marcel Verpaalen <marcel@verpaalen.com>

### CODE REVIEW

File: bundles/org.openhab.binding.miio/src/main/resources/database/philips.light.bulb.json [hunk 1]
```diff
@@ -105,20 +104,6 @@
...
-				"channel": "toggle",
-				"type": "Switch",
-				"refresh": false,
-				"ChannelGroup": "actions",
-				"actions": [
-					{
-						"command": "toggle",
-						"parameterType": "EMPTY"
-					}
-				]
```
Comment (cpmeister) [inline bundles/org.openhab.binding.miio/src/main/resources/database/philips.light.bulb.json:121]: Was this moved somewhere else?

File: bundles/org.openhab.binding.miio/README.md [hunk 2]
```diff
@@ -2275,13 +2265,12 @@ note: Autogenerated example. Replace the id (light) in the channel with your own
...
 Switch power "Power" (G_light) {channel="miio:basic:light:power"}
-Number brightness "Brightness" (G_light) {channel="miio:basic:light:brightness"}
-Number cct "Correlated Color Temperature" (G_light) {channel="miio:basic:light:cct"}
+Dimmer brightness "Brightness" (G_light) {channel="miio:basic:light:brightness"}
+Dimmer cct "Correlated Color Temperature" (G_light) {channel="miio:basic:light:cct"}
 Number scene "Scene" (G_light) {channel="miio:basic:light:scene"}
 Number dv "DV" (G_light) {channel="miio:basic:light:dv"}
 Switch switchscene "Switch Scene" (G_light) {channel="miio:basic:light:switchscene"}
 Switch delayoff "Delay Off" (G_light) {channel="miio:basic:light:delayoff"}
-Switch toggle "Toggle" (G_light) {channel="miio:basic:light:toggle"}
```
Comment (cpmeister) [inline bundles/org.openhab.binding.miio/README.md:2284]: why is toggle removed?

File: bundles/org.openhab.binding.miio/src/main/resources/database/philips.light.bulb.json (hunk 1 above)
Comment (marcelrv) [reply bundles/org.openhab.binding.miio/src/main/resources/database/philips.light.bulb.json:121]: I checked the latest versions of the apk files and the toggle command cannot be found anymore.
I tested it on my bulb and it did not react, so it seems it is removed from the device

Comment (marcelrv) [reply bundles/org.openhab.binding.miio/src/main/resources/database/philips.light.bulb.json:121]: I think it was always bit of an odd command... as the (mihome) app could not be sure what was the state after sending it... like if in the mean time the status on the device was already changed e.g. by light switch, the app could think it was switching it off whereas it then would in reality toggle it back on.

**Pull request merged**